extensions.json

```
@@ -1774,9 +1768,8 @@
     },
     {
       "id": "vscodevim.vim",
-      "download": "https://github.com/VSCodeVim/Vim/releases/download/v1.21.5/vim-1.21.5.vsix",
-      "version": "1.21.5",
-      "web": true
+      "download": "https://github.com/VSCodeVim/Vim/releases/download/v1.21.6/vim-1.21.6.vsix",
+      "version": "1.21.6"
```

Probably want to keep that.

```suggestion
      "version": "1.21.6",
      "web": true
```